pkg/agent/cniserver/ipam/antrea_ipam_controller.go

```diff
@@ -91,34 +117,100 @@ func (c *AntreaIPAMController) Run(stopCh <-chan struct{}) {
 	}()
 
 	klog.InfoS("Starting", "controller", controllerName)
-	if !cache.WaitForNamedCacheSync(controllerName, stopCh, c.namespaceInformer.Informer().HasSynced, c.ipPoolInformer.Informer().HasSynced) {
+	if !cache.WaitForNamedCacheSync(controllerName, stopCh, c.namespaceInformer.Informer().HasSynced, c.ipPoolInformer.Informer().HasSynced, c.podInformer.HasSynced) {
 		return
 	}
 
 	<-stopCh
 }
 
-func (c *AntreaIPAMController) getIPPoolsByNamespace(namespace string) []string {
+func (c *AntreaIPAMController) getIPPoolsByPod(namespace, name string) ([]string, []net.IP, *crdv1a2.IPAddressOwner, error) {
```

Comment:
namespace -> Namespace

Reply:
Checked other functions in Antrea, we always use `namespace` for property.

Reply:
You are right. My mistake here.